docs/customization/customizing-emails.mdx

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 6
+---
+
+# Customizing Transactional Emails
+
+This guide will teach you how to customize transactional emails provided by Solidus.
```

Comment:
Nothing inherently wrong with this line, but I think it doesn't play well with the rest of the guides, where we're usually jumping straight to the point :slightly_smiling_face:

Reply:
This is recommended by Diataxis, I think, so I'd be okay with keeping it (and gradually adding it to the new guides too!).

Reply:
Oh, ok. I feel it's a bit redundant with the title, though :slightly_smiling_face:

Reply:
I had the same feeling @waiting-for-dev, FWIW.